File: 2026/3 periodo/POO - PROG ORIENTADA A OBJETOS /05-03-2026/ATIVIDADE-M1/Main.java
import java.util.Scanner;

class IMC {
float calculoIMC(){

    float IMC;
    int peso;
    float altura;

    Scanner entrada = new Scanner(System.in);

    System.out.println("Digite o seu peso em valor inteiro:");
    peso = entrada.nextInt();

    System.out.println("Digite a sua altura em metros (ex: 1.75):");
    altura = entrada.nextFloat();

    IMC = peso / (altura * altura);

    if(IMC < 18.5){
        System.out.println("Abaixo do peso");
    }else if(IMC < 25){
        System.out.println("Peso normal");
    }else if(IMC < 30){
        System.out.println("Sobrepeso");
    }else if(IMC < 35){
        System.out.println("Obesidade grau 1");
    }else if(IMC < 40){
        System.out.println("Obesidade grau 2");
    }else{
        System.out.println("Obesidade grau 3");
    }

    return IMC;
}

}

public class Main{
public static void main(String[] args){

    IMC pessoa = new IMC(); 

    float resultado = pessoa.calculoIMC();  

    System.out.println("Seu IMC é: " + resultado);

}
    }

File: 2026/3 periodo/POO - PROG ORIENTADA A OBJETOS /05-03-2026/ATIVIDADE-M1/main.cpp
#include <iostream>
#include <stdio.h>


using namespace std;

float calculoIMC(){
    float IMC;
    int peso;
    float altura;
    cout<<"Digite o seu peso em valor inteiro: " <<endl;
    cin>> peso;
    cout<<"Digite a sua altura, em metros, exemplo 1.75m para equivalente a 175cm: " <<endl;
    cin>>altura;

    IMC = peso / (altura * altura);

    if(IMC < 18.5){
        cout<<"abaixo do peso: " <<endl;
    }else if(IMC > 18.5 && IMC < 24.9){
        cout<<"Peso normal: " <<endl;
    }else if(IMC > 25.0 && IMC < 29.9){
        cout<<"Sobrepeso: " <<endl;
    }else if(IMC > 30.0 && IMC < 34.9){
        cout<<"Obesidade grau 1: " <<endl;
    }else if(IMC > 35.0 && IMC < 39.9){
        cout<<"Obesiade grau 2: " <<endl;
    }else{
        cout<<"Obesidade grau 3: " <<endl;
    }

    return IMC;
}

int main(){
    float IMC = calculoIMC();
    cout<<" seu imc é: "<<IMC <<endl;
    return 0;
}